Practice3/Decision1.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Practice3
+{
+    class Decision1
+    {
+        List<int> _list = new List<int>();
+
+        public int MinRange { get; set; }
+        public int MaxRange { get; set; }
+
+        public void SetRange()
+        {
+            Console.WriteLine("Plase set min and max value that can be added into the array.");
+            Console.WriteLine("Min value:");
+            MinRange = Convert.ToInt32(Console.ReadLine());
+
+            Console.WriteLine("Max value:");
+            MaxRange = Convert.ToInt32(Console.ReadLine());
+        }
+
+        public void AddValues()
+        {
+            while (true)
+            {
+                Console.WriteLine($"Type a number between {MinRange} and {MaxRange} or click enter to see results");
+                String input = Console.ReadLine();
```

Comment:
I believe you wanted to use 'string' here (c# type, not String object type)

Reply:
Done. Yes, it is misprinted

Reply:
there's still "String"